lib/tapioca/gem/listeners/not_owned_constants.rb

```diff
@@ -0,0 +1,32 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Tapioca
+  module Gem
+    module Listeners
+      class NotOwnedConstants < Base
+        extend T::Sig
+
+        include Runtime::Reflection
+
+        private
+
+        sig { override.params(event: ScopeNodeAdded).void }
+        def on_scope(event)
+          mixin = event.constant
+
```

Comment:
I think we can return early if `mixin` is a `Class` instance, since no `Class` can be mixed into something else:
```suggestion
          mixin = event.constant

          return if Class === mixin # Classes can't be mixed into anything
```